src/generate/renders/generated/CliCustom.ts

```diff
@@ -209,7 +209,8 @@ function ConstructMethodBodyParameter(model: CodeModelAz, needGeneric = false, r
                         if (
                             originalParameterStack.last?.language?.['cli']?.[
                                 'moved-from-python'
-                            ] !== true
+                            ] !== true &&
```

Comment:
can we just use !originalParameterStack.last?.language......

Reply:
What's your concern ?